pyspades/server.py

```diff
@@ -159,6 +159,9 @@ def send_contained(self, contained, unsequenced=False, sender=None,
             else:
                 player.peer.send(0, packet)
 
+    # backwards compatability
+    send_contained = broadcast_contained
```

Comment:
might be nice to have a function that generates a wrapper that creates a DeprecationWarning when called in the future.

Reply:
That'd indeed be nice. Created an issue for it.